src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -98,6 +78,36 @@ public Command parseCommand(String userInput) throws ParseException {
         case ImportCommand.COMMAND_WORD:
             return new ImportCommandParser().parse(arguments);
 
+        default:
+            return parseTwoWordCommand(commandWord, arguments);
+        }
+    }
+
+    private Command parseTwoWordCommand(String commandWord, String arguments) throws ParseException {
+        boolean isTwoWordCommand = arguments.length() > 0
+                && !arguments.startsWith(" -") && !Character.isDigit(arguments.charAt(1));
+
+        if (isTwoWordCommand) {
+            commandWord = extractFullCommandWord(commandWord, arguments);
+            arguments = extractArguments(arguments);
+        } else {
+            throw new ParseException(MESSAGE_UNKNOWN_COMMAND);
+        }
+
```

Comment:
Should we do this check within the `default` case in the `parseCommand` method above? So this method will only take in valid two word commands and is only responsible for returning the correct parser.